src/MultiDropdown.js

```diff
@@ -2,21 +2,84 @@ import React, { Component } from "react";
 import PropTypes from "prop-types";
 import * as countriesCities from "countries-cities";
 import "./MultiDropdown.css";
-import _ from "lodash";
-
-// HINT User CustomDropdown.js here
-
+import CustomDropdown from "./CustomDropdown";
 export default class MultiDropdown extends Component {
   constructor(props) {
     super(props);
-    this.state = {};
+    this.state ={
+        "selectedValue1":null,
```

Comment:
You don't need to make these strings.

Reply:
Also why call this selectedValue1? You know it's a country, why not give it a better name?